packages/dm-core-plugins/src/form/templates/ArrayComplexTemplate.tsx

```diff
@@ -80,15 +80,15 @@ export const ArrayComplexTemplate = (props: TArrayTemplate) => {
             ))}
         </FormTemplate.Header.Actions>
       </FormTemplate.Header>
-      {canExpand && isExpanded && (
+      <div className={`${canExpand && isExpanded ? '' : 'hidden'}`}>
```

Comment:
Maybe FormTemplate.Content could take in a className or hidden field so that we add the class to the wrapper that's already there? So that we don't wrap it in a extra div when it's not needed 🙂

Reply:
True, good idea!

Reply:
will have a prop hidden: boolean

Reply:
or expanded prop

Reply:
Yeah, I guess default hidden and then having a visible prop also makes sense - more reflects our checks and default functionality. Just do what looks good to you 👍🏻

Reply:
also need to make sure it does not load things that cannot be expanded .. hm

Reply:
perhaps we should differentiate between canExpand, and isExpanded.

Reply:
Ah, yes! Maybe we can still wrap the ViewCreator with canExpand? 'cause that doesn't change on expand/hide. We don't want to render the ViewCreator in that instance anyways

Reply:
So pretty much what you just implemented 😆 👍🏻

Reply:
yes!